pkg/reconciler/mtbroker/trigger/controller.go

```diff
@@ -76,19 +79,8 @@ func NewController(
 	brokerInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
 		FilterFunc: brokerFilter,
 		Handler: controller.HandleAll(func(obj interface{}) {
-
-			if broker, ok := obj.(*eventingv1.Broker); ok {
```

Comment:
+1 on moving this out and making it testable. We should use this as a pattern to help unit test a lot of where we do this. Might make sense for someone to make good-first-issues after a scan of the repo. I can help with that.

Reply:
Ack 👍